requirements.prod.in

```diff
@@ -1,3 +1,8 @@
-fastapi
-uvicorn
 pyoxidizer
+uvicorn
+django==4.1
```

Comment:
Can we use the magic symbol that gives us the latest bugfix of 4.1, whatever that happens to be? I always forget what that magic symbol is…

Reply:
I didn't realise pip-tools supported that!

Reply:
Oh I might be mis-remembering.  I _thought_ it supported any version spec pip did because it used pip under the hood for it but it's also been ages.  This is also absolute fine, my goal here was just to make sure we get bug fix versions 👍